media/CircleEditor/view.js

```diff
@@ -821,6 +825,22 @@ view.View = class {
                     if (this._page !== 'default') {
                         this.show('default');
                     }
+                    for (let idx = 0; idx < model.graphs.length; idx++) {
+                        model._graphs[idx]['_subgraphIdx'] = idx;
+                        for (let jdx = 0; jdx < model.graphs[idx].nodes.length; jdx++) {
+                            model._graphs[idx]._nodes[jdx]['_subgraphIdx'] = idx;
+                            if (model._graphs[idx]._nodes[jdx]._type?.category === 'custom') {
+                                model._graphs[idx]._nodes[jdx]._isCustom = true;
+                                vscode.postMessage({
+                                    command: 'getCustomOpAttrT',
```

Comment:
I am little bit confused...
It seems that there are two `getCustomOpAttrT`, which are in `index.js` and `CircleEditorProvider.ts`.
Then the two message listeners are really for **getting** custom operator attributes type? not **setting**?
Furthermore, there is no `CircleEditorProvider.ts` to handle this message...

Reply:
I'll change `get-` to `set-` in `index.js`. 
Would it be better to upload the related parts of the message after `CircleEditorProvider.ts` is uploaded? 
I think that sending and receiving messages is always complementary, so what should we do with the commands that already exist?

Reply:
> Would it be better to upload the related parts of the message after CircleEditorProvider.ts is uploaded?

Leaving it as TODO and then implementing after message listeners in `CircleEditorProvider.ts` is introduced would be better :)

> so what should we do with the commands that already exist?

What does this mean...? T.T

Reply:
I left the related part as TODO..